core/test/functional/api/v0.1/public_api_spec.js

```diff
@@ -77,7 +77,8 @@ describe('Public API', function () {
             });
     });
 
-    it('browse pages', function (done) {
+    // TODO: move to v2 folder, and don't skip once we have auth helper for content api tests
+    it.skip('browse pages', function (done) {
```

Comment:
@gargol is working on that

Reply:
nice :+1:

Reply:
@allouis this went out with 76f4a4bb036405bfc96db5a6f74a9a1103523303 ;)